roles/content_views/README.md

```diff
@@ -0,0 +1,64 @@
+theforeman.foreman.content_views
+=========================================
+
+This role creates and manages Content Views.
+
+Role Variables
+--------------
+
+This role supports the [Common Role Variables](https://github.com/theforeman/foreman-ansible-modules/blob/develop/README.md#common-role-variables).
+
+The main data structure for this role is the list of `foreman_content_views`. Each Content View requires the following fields:
+
+- `name`
+- `content_view`
+- `content_view_update`
+- `repos`
+  - `name`
+  - `product`
+  - `basearch`: (optional)
+  - `releasever`: (optional)
+
+The following fields are optional and will be omitted by default:
+
+- `filters`
+
+### Secure Logging Variables
+The following Variables compliment each other.
+If both variables are not set, secure logging defaults to false.
+The role defaults to False as normally the add content view task does not include sensitive information.
+`foreman_configuration_content_views_secure_logging` defaults to the value of `foreman_configuration_secure_logging` if it is not explicitly called. This allows for secure logging to be toggled for the entire suite of configuration roles with a single variable, or for the user to selectively use it.
+
+|Variable Name|Default Value|Required|Description|
+|:---:|:---:|:---:|:---:|
+|`foreman_configuration_content_views_secure_logging`|`False`|no|Whether or not to include the sensitive content view role tasks in the log.  Set this value to `True` if you will be providing your sensitive values from elsewhere.|
+|`forman_configuration_secure_logging`|`False`|no|This variable enables secure logging as well, but is shared across multiple roles, see above.|
+
+Example Playbooks
+-----------------
+
```

Comment:
Could you add here a brief description of what this example playbook does? As it gets expanded to include multiple examples, it's always useful to have these so that users looking to quickly get started can find one which most closely matches their use case and modify accordingly.